test/rulesets/Base/mapreduce.jl

```diff
@@ -2,39 +2,23 @@
     @testset "sum" begin
         sizes = (3, 4, 7)
         @testset "dims = $dims" for dims in (:, 1)
-            fkwargs = (dims=dims,)
             @testset "Array{$N, $T}" for N in eachindex(sizes), T in (Float64, ComplexF64)
                 s = sizes[1:N]
                 x = randn(T, s...)
```

Comment:
since only using `s` once:
```suggestion
                x = randn(T, sizes[1:N]...)
```